eks-anywhere-snow/Addons/Partner/Sysdig/sysdig.yaml

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: sysdig
+  namespace: sysdig
+spec:
+  chart:
+    spec:
+      chart: sysdig-deploy
+      reconcileStrategy: ChartVersion
+      sourceRef:
+        kind: HelmRepository
+        name: sysdig
+        namespace: flux-system
+      version: 1.5.46
+  interval: 30s
+  url: https://charts.sysdig.com
```

Comment:
Why do we need `url` here if we already HelmRepostiory pointing to that url ?

Reply:
you are right, thanks

Reply:
Sounds good.